CHANGELOG.md

```diff
@@ -2,15 +2,21 @@
 
 This file is used to list changes made in each version of the Java cookbook.
 
-## Unreleased
+## 8.0.0 (tbc)
```

Comment:
Can you create an `UPGRADING.md` as well to help people out?

Reply:
UPGRADING, throw it all out and start again? 😬